src/test/java/com/puppycrawl/tools/checkstyle/utils/CommonUtilTest.java

```diff
@@ -364,6 +369,13 @@ public void testIsBlank() {
         assertFalse(CommonUtil.isBlank("string"), "Should return false when string is not empty");
     }
 
+    @Test
+    public void testIsBlankCodePoint() {
+        final int[] input = {115, 116, 114, 105, 110, 103};
```

Comment:
Please make all code point array initializations in tests like:

```
      ... =  "my string here".codePoints().toArray();
```

Tests also serve as documentation; this will make these tests much easier to read (and write). 

This should help with https://github.com/checkstyle/checkstyle/pull/10976#issuecomment-997161921 also.

Reply:
ok